src/inputs/WizFormGroup.tsx

```diff
@@ -3,10 +3,16 @@ import { ReactNode } from 'react'
 import { LabelHelp } from '../components/LabelHelp'
 import { InputCommonProps, useID, useInputValidation } from './Input'
 
-type WizFormGroupProps = InputCommonProps & { children: ReactNode; isHelperTextBeforeField?: boolean }
+type WizFormGroupProps = InputCommonProps & {
+    children: ReactNode
+    isHelperTextBeforeField?: boolean
+    helperTextInvalid?: (error: string | undefined) => ReactNode
```

Comment:
Is it necessary to expose this property?

Reply:
I see why it is exposed now, but it seems odd to me to combine the validation text with the link out... if the field is required, then no item is selected, so we shouldn't be showing a link in that case.

Reply:
Agreed. I will revert that change.